swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
 class Item:
-    pass
+    DEFAULT_ITEM_AGE = 3
+    # Note about my using 2 different patterns for making the child classes (Clothing, Decor, Electronics) use the 
+    # parent class' default values for condition and age. In production code, I'd use the same pattern for all of these cases 
+    # to make the code more consistent and readbale. Here, I'm using 2 different patterns for my own laerning/practice and 
+    # to have both as examples to look back on.  
+    
+    def __init__(self, category = '', condition = 0, age = DEFAULT_ITEM_AGE):
+        self.category = category
+        self.condition = condition
+        self.age = age   
+
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+         descriptions = ["Uh, ya sure you want this?",
+                "Remember, all swaps final",
+                "Still works",
+                "Eh, good enough!",
+                "Good enough for company",
+                "P-r-i-m-o!"                
+         ]
+         return descriptions[self.condition]
```

Comment:
Like we discussed in class, how would you handle floats?

Reply:
I also like these descriptions 😂

Reply:
I'd return descriptions[int(self.condition)] or import math and use math.floor 
return descriptions[math.floor(self.condition)] , which feels a little more descriptive